rust/sdk-processor/src/steps/common/processor_status_saver.rs

```diff
@@ -33,15 +34,22 @@ pub fn get_processor_status_saver(
         }
```

Comment:
I'm uncertain if we should proceed this way since the behavior is identical. I was just reusing this, but I'm not sure if that might confuse people.

Reply:
hmm, don't we use alias for backfilling?

Reply:
Oh my bad, that's right - you could set the backfill_alias to anything